modules/standard/Path.chpl

```diff
@@ -260,5 +260,32 @@ proc file.realPath(): string throws {
       }
     }
    return result;
- }  
+ }
+ 
+/* Determines whether the path specified is an absolute path in Unix
+   environment and returns error if it is not in Unix environment
+
+   :arg name: the path to be checked.
+   :type name: `string`
+
+   :return: `true` if `name` is an absolute path, `false` otherwise
+   :rtype: `bool`
+*/
+
+  proc isAbsPath(name: string): bool {
+     if (CHPL_TARGET_PLATFORM == 'linux64' || CHPL_TARGET_PLATFORM == 'linux32' || CHPL_TARGET_PLATFORM == 'darwin') {
```

Comment:
This line is likely longer than 80 characters, I would recommend putting each part of the conditional on a separate line:
```chapel
if (CHPL_TARGET_PLATFORM == 'linux64' ||
    CHPL_TARGET_PLATFORM == 'linux32' ||
    CHPL_TARGET_PLATFORM == 'darwin') {
```